package.json

```diff
@@ -5,7 +5,7 @@
   "description": "",
   "author": "David J. Hamilton <davidjh@hjdivad.com>",
   "scripts": {
-    "build": "tsc --build",
+    "build": "tsc --build && cp src/check-suite-complete/function.json check-suite-complete/",
```

Comment:
😭, I wonder if I can make TS copy this for us 🤔

Reply:
We can also just check it in at the dist location since it isn't actually transformed.  Just need a gitignore exception